pyproject.toml

```diff
@@ -80,21 +67,13 @@ Markdown = "*"
 # Render custom markdown for version added/changed/remove notes
 markdown-version-annotations = "1.0.1"
 # Rendering docs to HTML
-mkdocs = "1.5.2"
-mkdocs-include-markdown-plugin = "6.0.3"
```

Comment:
I still want to get this one into the cookie, I think it is a useful plugin to have.